docs/configuration/scheduling.rst

```diff
@@ -50,9 +56,9 @@ Then, you can use Airflow's data-aware scheduling capabilities to schedule ``my_
 
     project_two = DbtDag(
         # for airflow <=2.3
-        # schedule=[get_dbt_dataset("my_conn", "project_one", "my_model")],
+        # schedule_interval=[Dataset("postgres://host:5432/database.schema.my_model")],,
```

Comment:
appears that Data-aware scheduling using Datasets was only introduced starting Airflow 2.4. So Airflow <=2.3 might not support this(?)

Ref: https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/datasets.html

Reply:
Well spotted, Pankaj, just fixed!